components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FormattingUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.gradle.enterprise.cli;
+
+import java.time.Duration;
+import java.util.Locale;
+
+final class FormattingUtils {
```

Comment:
I would highly favor that we do not use Utils in class names. `Formatting` or `DurationFormat` (preferred) would be better.

Reply:
Please also adjust the test class name similarly.